packages/web3-eth-miner/package.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "web3-eth-miner",
+    "namespace": "ethereum",
+    "version": "1.0.0-beta.52",
+    "description": "Web3 module to interact with the Ethereum blockchain accounts stored in the node.",
+    "repository": "https://github.com/ethereum/web3.js/tree/1.0/packages/web3-eth-miner",
+    "license": "LGPL-3.0",
+    "main": "dist/web3-eth-miner.cjs.js",
+    "module": "dist/web3-eth-miner.esm.js",
+    "browser": "dist/web3-eth-miner.umd.js",
+    "scripts": {
+        "build": "rollup -c",
+        "dev": "rollup -c -w",
+        "test": "jest",
+        "dtslint": "dtslint types --onlyTestTsNext"
+    },
+    "types": "types/index.d.ts",
+    "dependencies": {
+        "@babel/runtime": "^7.3.1",
+        "web3-core": "1.0.0-beta.52",
+        "web3-core-helpers": "1.0.0-beta.52",
+        "web3-core-method": "1.0.0-beta.52",
+        "web3-net": "1.0.0-beta.52",
+        "web3-providers": "1.0.0-beta.52",
+        "web3-utils": "1.0.0-beta.52"
+    },
+    "devDependencies": {
+        "definitelytyped-header-parser": "^1.0.1",
+        "dtslint": "0.4.2",
+        "web3-eth-accounts": "1.0.0-beta.52"
```

Comment:
you cant add this as a `dev` dependency if you got it in the typing's, it has to be a a proper dependency as NPM wont publish that on install. If i was you in your typing's i would just make it a `any` to avoid adding a dependency.